frontend/src/styles/_uswds-theme.scss

```diff
@@ -48,6 +49,8 @@ in the form $setting: value,
     hover: false,
     visited: false
   ),
+  $height-settings: (responsive: true),
```

Comment:
Need responsiveness for the height classes so that the logo can be different sizes at different breakpoints.